doc/lightning-listtransactions.7.md

```diff
@@ -26,21 +26,21 @@ RETURN VALUE
 
 On success, the command will return a list of transactions, each object represents a transaction, with the following details:
 
-- *hash*: A string that represents the hash of transaction, which the caller can use to find it on the blockchain.
```

Comment:
You're stripping the type information from these descriptions. NACK.

Reply:
The types were wrong in many places. They said "a string that represents whatever" a million times, but very often the type was actually a number. And there was no reference whatsoever to what was the encoding or format of the "string".

what I tried to do was to make the descriptions more readable and more helpful.

I actually think I should have removed more of these type information, because there are examples right below these descriptions and in the examples the type is much clearer.

Reply:
How can a "hex-encoded something" not be a string?

Reply:
> The types were wrong in many places. 

If the types are wrong, they should be fixed.

> but not necessarily containing all the possible details.

The point of documentation is to be both clear and contain all relevant details.

Reply:
The relevant details are in the examples.